### PR TITLE
Apply dacx updates

### PR DESCRIPTION
**Pull request by flossypurse**

### CODE REVIEW

File: docs-src/python/how-to-list-workflow-executions-using-the-client-in-python.md [hunk 1]
```diff
@@ -2,16 +2,15 @@
 id: how-to-list-workflow-executions-using-the-client-in-python
 title: How to list Workflows in Python
 sidebar_label: List Workflows
-description: List Workflows from the Client using `list_workflows()`.
-tags:
-  - developer-guide
-  - sdk
-  - python
+description: To list Workflows from the Client, use the [list_workflows()](https
```
Comment (djmagee) [inline docs-src/python/how-to-list-workflow-executions-using-the-client-in-python.md:5]:
```suggestion
description: To list Workflows from the Client, use the list_workflows() method on the Client handle.
```

File: docs-src/python/how-to-log-from-a-workflow-in-python.md [hunk 2]
```diff
@@ -2,43 +2,22 @@
 id: how-to-log-from-a-workflow-in-python
 title: How to log from a Workflow in Python
 sidebar_label: Log from a Workflow
-description: Log from a Workflow
-tags:
-  - developer-guide
-  - sdk
-  - python
+description: To log from a Workflow in Python, import the logging module `logging` and set your logging configuration to a level you want to expose logs to. Then in your Workflow, set your [`logger`](https
```
Comment (djmagee) [inline docs-src/python/how-to-log-from-a-workflow-in-python.md:5]:
```suggestion
description: To log from a Workflow in Python, import the logging module `logging` and set your logging configuration to a level you want to expose logs to.
```

File: docs-src/python/how-to-log-from-a-workflow-in-python.md [hunk 3]
```diff
@@ -2,43 +2,22 @@
...
 ---
 
-You can log from a Workflow using Python's standard library, by importing the logging module `import logging`.
+You can log from a Workflow using Python's standard library, by importing the logging module `logging`.
```
Comment (djmagee) [inline docs-src/python/how-to-log-from-a-workflow-in-python.md:8]:
```suggestion
You can log from a Workflow using Python's standard library by importing the logging module `logging`.
```

File: docs-src/python/how-to-log-from-a-workflow-in-python.md [hunk 4]
```diff
@@ -2,43 +2,22 @@
...
 
 Set your logging configuration to a level you want to expose logs to.
-The following example sets the logging information level to `INFO`.
 
 ```python
 logging.basicConfig(level=logging.INFO)
 ```
 
 Then in your Workflow, set your [`logger`](https://python.temporal.io/temporalio.workflow.html#logger) and level on the Workflow. The following example logs the Workflow.
```
Comment (djmagee) [inline docs-src/python/how-to-log-from-a-workflow-in-python.md:16]:
```suggestion
Then in your Workflow, set your [`logger`](https://python.temporal.io/temporalio.workflow.html#logger) and level on the Workflow.
The following example logs the Workflow.
```

**Comment by flossypurse** [comment]
Most of this was applied in another PR. We would need to run the tool fresh to see what we have.

**Pull request closed without merging**